src/cbmc/cbmc_parse_options.cpp

```diff
@@ -239,23 +243,29 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
   if(cmdline.isset("cover"))
     options.set_option("unwinding-assertions", false);
   else
-    options.set_option("unwinding-assertions",
+  {
+    options.set_option(
+      "unwinding-assertions",
       cmdline.isset("unwinding-assertions"));
+  }
 
   // generate unwinding assumptions otherwise
-  options.set_option("partial-loops",
-   cmdline.isset("partial-loops"));
+  options.set_option(
+    "partial-loops",
+    cmdline.isset("partial-loops"));
 
   if(options.get_bool_option("partial-loops") &&
      options.get_bool_option("unwinding-assertions"))
   {
-    error() << "--partial-loops and --unwinding-assertions must not be given together" << eom;
+    error() << "--partial-loops and --unwinding-assertions ";
+    error() << "must not be given together" << eom;
```

Comment:
Already applied in #343 https://github.com/diffblue/cbmc/pull/343/files#diff-41db988013f70caad45ce489749c7df6R259